### PR TITLE
ESBJAVA-4640: adding google-libphonenumber orbit

### PR DESCRIPTION
**Pull request by nuwanw**

### CODE REVIEW

File: google-libphonenumber/7.4.2.wso2v1/pom.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Export-Package>
+                            com.google.i18n.phonenumbers.*;version="${project.version}"
```
Comment (Niranjan-K) [inline google-libphonenumber/7.4.2.wso2v1/pom.xml]: The export version should be without the suffix wso2v1

File: google-libphonenumber/7.4.2.wso2v1/pom.xml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                        <Private-Package>
+                        </Private-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+        <version.com.googlecode.libphonenumber>7.4.2</version.com.googlecode.libphonenumber>
```
Comment (Niranjan-K) [inline google-libphonenumber/7.4.2.wso2v1/pom.xml]: As a general practice, the version is mentioned at the end of the property. Shall we follow the same approach as https://github.com/wso2/orbit/pull/190/files

File: google-libphonenumber/7.4.2.wso2v1/pom.xml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+ ~ limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.wso2.orbit.com.googlecode.libphonenumber</groupId>
+    <artifactId>com.googlecode.libphonenumber</artifactId>
```
Comment (Niranjan-K) [inline google-libphonenumber/7.4.2.wso2v1/pom.xml]: This should be the same as the artifactId, i.e., libphonenumber

File: google-libphonenumber/7.4.2.wso2v1/pom.xml (hunk 2 above)
Comment (nuwanw) [inline google-libphonenumber/7.4.2.wso2v1/pom.xml]: Fixed

File: google-libphonenumber/7.4.2.wso2v1/pom.xml (hunk 3 above)
Comment (nuwanw) [inline google-libphonenumber/7.4.2.wso2v1/pom.xml]: Fixed

**Pull request merged**